.github/workflows/ci.yml

```diff
@@ -27,7 +27,6 @@ jobs:
           - x64
           - x86
         python:
-          - cp35-cp35m
```

Comment:
README.md also needs to be changed to say python 3.6+ is required

Reply:
Odd Y versions in Python are experimental. Maybe we should stick to even Y versions for requirements.
I mean, now we say 3.6+, next time 3.8+, next time 3.10+, no stops in the middle.